meta-mel/meta-altera/recipes-multimedia/mplayer/mpv_0.15.0.bb

```diff
@@ -20,8 +20,6 @@ SRC_URI[waf.sha256sum] = "01bf2beab2106d1558800c8709bc2c8e496d3da4a2ca343fe091f2
 
 inherit pkgconfig
 
-# Note: both lua and libass are required to get on-screen-display (controls)
-PACKAGECONFIG ??= "lua libass"
```

Comment:
Rather than doing it this way, why not do a PACKAGECONFIG_remove_cyclone5 through a bbappend perhaps just to make sure we can use this stuff on other BSPs with controls enabled if needed.